src/main/webapp/resources/js/pages/admin/components/AdminSideMenu.jsx

```diff
@@ -43,7 +43,23 @@ export default function AdminHeader() {
                 <img style={{ height: 64, width: 200 }} src={setBaseUrl("/resources/img/irida_logo_dark.svg")}/>
               </Link>
               <Menu className={"t-admin-side-menu"} style={{ height: '100vh' }} theme={"dark"} mode={"inline"}
-                    selectedKeys={[keyname ? parseKey(keyname) : ADMIN.USERS]}>
+                    selectedKeys={[keyname ?
+                      ( parseKey(keyname) !== "admin" ?
+                          !statsPages.includes(parseKey(keyname)) ?
```

Comment:
Is this still needed after the way I refactored it?

Reply:
If we remove it then if one of the tiny chart links are clicked on then the menu item is no longer highlighted

Reply:
This should be fixed in the code I just sent you.